meta-ibm/recipes-phosphor/inventory/phosphor-inventory-manager/obmc-clear-all-faults@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Clears all fault LED to %i
+Wants=xyz.openbmc_project.Inventory.Manager
+After=xyz.openbmc_project.Inventory.Manager
+After=op-reset-chassis-running@0.service
```

Comment:
Better to use the generic target in case the service changes. So after the action target and after the synchronization target within the action.

After=obmc-chassis-powerreset@0.target
Wants=obmc-power-reset-on@0.target
After=obmc-power-reset-on@0.target